### ISSUE TITLE
Nested virt detection failing on AMD

### ISSUE THREAD

**Issue by wainersm** [karmab/kcli#667]
I'm getting the `Hypervisor not compatible with nesting. Skipping` warning although I've nested virt proper enabled on my workstation.

Machine:
* CPU model: AMD Ryzen 7 PRO 7840HS w/ Radeon 780M Graphics
* Linux: Fedora 29
* Packages:
  * qemu-kvm-8.1.3-5.fc39.x86_64
  * libvirt-9.7.0-3.fc39.x86_64
  * kcli-99.0.0.git.202404191218.5eb02fd-0.fc39.x86_64

qemu-kvm, libvirt and kvm nested virt are proper set:
```
$ cat /sys/module/kvm_amd/parameters/nested
1
$ systool -m kvm_amd -v   | grep -i nested
    nested              = "1"
```

The CPU flags reports `svm`:
```
$ cat /proc/cpuinfo | grep ^flags | head -1 | grep svm
flags		: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx mmxext fxsr_opt pdpe1gb rdtscp lm constant_tsc rep_good amd_lbr_v2 nopl nonstop_tsc cpuid extd_apicid aperfmperf rapl pni pclmulqdq monitor ssse3 fma cx16 sse4_1 sse4_2 x2apic movbe popcnt aes xsave avx f16c rdrand lahf_lm cmp_legacy svm extapic cr8_legacy abm sse4a misalignsse 3dnowprefetch osvw ibs skinit wdt tce topoext perfctr_core perfctr_nb bpext perfctr_llc mwaitx cpb cat_l3 cdp_l3 hw_pstate ssbd mba perfmon_v2 ibrs ibpb stibp ibrs_enhanced vmmcall fsgsbase bmi1 avx2 smep bmi2 erms invpcid cqm rdt_a avx512f avx512dq rdseed adx smap avx512ifma clflushopt clwb avx512cd sha_ni avx512bw avx512vl xsaveopt xsavec xgetbv1 xsaves cqm_llc cqm_occup_llc cqm_mbm_total cqm_mbm_local user_shstk avx512_bf16 clzero irperf xsaveerptr rdpru wbnoinvd cppc arat npt lbrv svm_lock nrip_save tsc_scale vmcb_clean flushbyasid decodeassists pausefilter pfthreshold v_vmsave_vmload vgif x2avic v_spec_ctrl vnmi avx512vbmi umip pku ospke avx512_vbmi2 gfni vaes vpclmulqdq avx512_vnni avx512_bitalg avx512_vpopcntdq rdpid overflow_recov succor smca flush_l1d
```

The problem is that kcli relies on the XML output of libvirt's capabilities (e.g. `virsh capabilities`)  which **doesn't** report svm flag in the list of features. Initially I thought it could be a bug on Libvirt then I asked the experts on our internal virt channel in Slack; I was told that `virsh capabilities` doesn't report all features included on the CPU model but rather the list of additional features, thus if we can check all features included in a CPU model then we should use the `cpu-baseline` command (e.g.  `virsh capabilities | virsh cpu-baseline --features /dev/stdin`).

See the differences of output with and without cpu-baseline:
```
$ virsh capabilities | grep svm
      <feature name='svm-lock'/>
      <feature name='svme-addr-chk'/>
$ virsh capabilities | virsh cpu-baseline --features /dev/stdin | grep svm
  <feature policy='require' name='svm'/>
  <feature policy='require' name='svm-lock'/>
  <feature policy='require' name='svme-addr-chk'/>
```

The full output of capabilities:
```
<capabilities>

  <host>
    <uuid>ff1384c5-7c63-40f5-9d79-4be8b7893ebd</uuid>
    <cpu>
      <arch>x86_64</arch>
      <model>EPYC-Rome</model>
      <vendor>AMD</vendor>
      <microcode version='175128835'/>
      <signature family='25' model='116' stepping='1'/>
      <counter name='tsc' frequency='3792840000' scaling='yes'/>
      <topology sockets='1' dies='1' cores='8' threads='2'/>
      <maxphysaddr mode='emulate' bits='48'/>
      <feature name='ht'/>
      <feature name='monitor'/>
      <feature name='x2apic'/>
      <feature name='osxsave'/>
      <feature name='erms'/>
      <feature name='invpcid'/>
      <feature name='cmt'/>
      <feature name='avx512f'/>
      <feature name='avx512dq'/>
      <feature name='avx512ifma'/>
      <feature name='avx512cd'/>
      <feature name='avx512bw'/>
      <feature name='avx512vl'/>
      <feature name='avx512vbmi'/>
      <feature name='pku'/>
      <feature name='ospke'/>
      <feature name='avx512vbmi2'/>
      <feature name='gfni'/>
      <feature name='vaes'/>
      <feature name='vpclmulqdq'/>
      <feature name='avx512vnni'/>
      <feature name='avx512bitalg'/>
      <feature name='avx512-vpopcntdq'/>
      <feature name='flush-l1d'/>
      <feature name='avx512-bf16'/>
      <feature name='xsaves'/>
      <feature name='mbm_total'/>
      <feature name='mbm_local'/>
      <feature name='cmp_legacy'/>
      <feature name='extapic'/>
      <feature name='ibs'/>
      <feature name='skinit'/>
      <feature name='wdt'/>
      <feature name='tce'/>
      <feature name='topoext'/>
      <feature name='perfctr_nb'/>
      <feature name='invtsc'/>
      <feature name='ibrs'/>
      <feature name='stibp-always-on'/>
      <feature name='amd-ssbd'/>
      <feature name='amd-psfd'/>
      <feature name='lbrv'/>
      <feature name='svm-lock'/>
      <feature name='tsc-scale'/>
      <feature name='vmcb-clean'/>
      <feature name='flushbyasid'/>
      <feature name='decodeassists'/>
      <feature name='pause-filter'/>
      <feature name='pfthreshold'/>
      <feature name='v-vmsave-vmload'/>
      <feature name='vgif'/>
      <feature name='vnmi'/>
      <feature name='svme-addr-chk'/>
      <feature name='no-nested-data-bp'/>
      <feature name='lfence-always-serializing'/>
      <feature name='null-sel-clr-base'/>
      <feature name='auto-ibrs'/>
      <pages unit='KiB' size='4'/>
      <pages unit='KiB' size='2048'/>
      <pages unit='KiB' size='1048576'/>
    </cpu>
    <power_management>
      <suspend_mem/>
    </power_management>
    <iommu support='yes'/>
    <migration_features>
      <live/>
      <uri_transports>
        <uri_transport>tcp</uri_transport>
        <uri_transport>rdma</uri_transport>
      </uri_transports>
    </migration_features>
    <topology>
      <cells num='1'>
        <cell id='0'>
          <memory unit='KiB'>64498412</memory>
          <pages unit='KiB' size='4'>16124603</pages>
          <pages unit='KiB' size='2048'>0</pages>
          <pages unit='KiB' size='1048576'>0</pages>
          <distances>
            <sibling id='0' value='10'/>
          </distances>
          <cpus num='16'>
            <cpu id='0' socket_id='0' die_id='0' core_id='0' siblings='0-1'/>
            <cpu id='1' socket_id='0' die_id='0' core_id='0' siblings='0-1'/>
            <cpu id='2' socket_id='0' die_id='0' core_id='1' siblings='2-3'/>
            <cpu id='3' socket_id='0' die_id='0' core_id='1' siblings='2-3'/>
            <cpu id='4' socket_id='0' die_id='0' core_id='2' siblings='4-5'/>
            <cpu id='5' socket_id='0' die_id='0' core_id='2' siblings='4-5'/>
            <cpu id='6' socket_id='0' die_id='0' core_id='3' siblings='6-7'/>
            <cpu id='7' socket_id='0' die_id='0' core_id='3' siblings='6-7'/>
            <cpu id='8' socket_id='0' die_id='0' core_id='4' siblings='8-9'/>
            <cpu id='9' socket_id='0' die_id='0' core_id='4' siblings='8-9'/>
            <cpu id='10' socket_id='0' die_id='0' core_id='5' siblings='10-11'/>
            <cpu id='11' socket_id='0' die_id='0' core_id='5' siblings='10-11'/>
            <cpu id='12' socket_id='0' die_id='0' core_id='6' siblings='12-13'/>
            <cpu id='13' socket_id='0' die_id='0' core_id='6' siblings='12-13'/>
            <cpu id='14' socket_id='0' die_id='0' core_id='7' siblings='14-15'/>
            <cpu id='15' socket_id='0' die_id='0' core_id='7' siblings='14-15'/>
          </cpus>
        </cell>
      </cells>
    </topology>
    <cache>
      <bank id='0' level='3' type='both' size='16' unit='MiB' cpus='0-15'/>
    </cache>
    <secmodel>
      <model>selinux</model>
      <doi>0</doi>
      <baselabel type='kvm'>system_u:system_r:svirt_t:s0</baselabel>
      <baselabel type='qemu'>system_u:system_r:svirt_tcg_t:s0</baselabel>
    </secmodel>
  </host>

  <guest>
    <os_type>hvm</os_type>
    <arch name='i686'>
      <wordsize>32</wordsize>
      <emulator>/usr/bin/qemu-system-i386</emulator>
      <machine maxCpus='255'>pc-i440fx-8.1</machine>
      <machine canonical='pc-i440fx-8.1' maxCpus='255'>pc</machine>
      <machine maxCpus='288'>pc-q35-5.2</machine>
      <machine maxCpus='255'>pc-i440fx-2.12</machine>
      <machine maxCpus='255'>pc-i440fx-2.0</machine>
      <machine maxCpus='1'>xenpv</machine>
      <machine maxCpus='255'>pc-i440fx-6.2</machine>
      <machine maxCpus='288'>pc-q35-4.2</machine>
      <machine maxCpus='255'>pc-i440fx-2.5</machine>
      <machine maxCpus='255'>pc-i440fx-4.2</machine>
      <machine maxCpus='255'>pc-i440fx-5.2</machine>
      <machine maxCpus='255' deprecated='yes'>pc-i440fx-1.5</machine>
      <machine maxCpus='255'>pc-q35-2.7</machine>
      <machine maxCpus='288'>pc-q35-7.1</machine>
      <machine maxCpus='255'>pc-i440fx-2.2</machine>
      <machine maxCpus='1024'>pc-q35-8.1</machine>
      <machine canonical='pc-q35-8.1' maxCpus='1024'>q35</machine>
      <machine maxCpus='255'>pc-i440fx-2.7</machine>
      <machine maxCpus='288'>pc-q35-6.1</machine>
      <machine maxCpus='128'>xenfv-3.1</machine>
      <machine canonical='xenfv-3.1' maxCpus='128'>xenfv</machine>
      <machine maxCpus='255'>pc-q35-2.4</machine>
      <machine maxCpus='255'>pc-i440fx-7.1</machine>
      <machine maxCpus='288'>pc-q35-2.10</machine>
      <machine maxCpus='1'>x-remote</machine>
      <machine maxCpus='288'>pc-q35-5.1</machine>
      <machine maxCpus='255' deprecated='yes'>pc-i440fx-1.7</machine>
      <machine maxCpus='288'>pc-q35-2.9</machine>
      <machine maxCpus='255'>pc-i440fx-2.11</machine>
      <machine maxCpus='288'>pc-q35-3.1</machine>
      <machine maxCpus='255'>pc-i440fx-6.1</machine>
      <machine maxCpus='288'>pc-q35-4.1</machine>
      <machine maxCpus='255'>pc-i440fx-2.4</machine>
      <machine maxCpus='255'>pc-i440fx-4.1</machine>
      <machine maxCpus='255'>pc-i440fx-5.1</machine>
      <machine maxCpus='255'>pc-i440fx-2.9</machine>
      <machine maxCpus='1'>isapc</machine>
      <machine maxCpus='255' deprecated='yes'>pc-i440fx-1.4</machine>
      <machine maxCpus='255'>pc-q35-2.6</machine>
      <machine maxCpus='255'>pc-i440fx-3.1</machine>
      <machine maxCpus='288'>pc-q35-2.12</machine>
      <machine maxCpus='288'>pc-q35-7.0</machine>
      <machine maxCpus='255'>pc-i440fx-2.1</machine>
      <machine maxCpus='288'>pc-q35-8.0</machine>
      <machine maxCpus='255'>pc-i440fx-8.0</machine>
      <machine maxCpus='288'>pc-q35-6.0</machine>
      <machine maxCpus='255'>pc-i440fx-2.6</machine>
      <machine maxCpus='288'>pc-q35-4.0.1</machine>
      <machine maxCpus='255'>pc-i440fx-7.0</machine>
      <machine maxCpus='255' deprecated='yes'>pc-i440fx-1.6</machine>
      <machine maxCpus='288'>pc-q35-5.0</machine>
      <machine maxCpus='288'>pc-q35-2.8</machine>
      <machine maxCpus='255'>pc-i440fx-2.10</machine>
      <machine maxCpus='288'>pc-q35-3.0</machine>
      <machine maxCpus='288'>pc-q35-7.2</machine>
      <machine maxCpus='288'>pc-q35-4.0</machine>
      <machine maxCpus='128'>xenfv-4.2</machine>
      <machine maxCpus='255'>pc-i440fx-6.0</machine>
      <machine maxCpus='288'>microvm</machine>
      <machine maxCpus='255'>pc-i440fx-2.3</machine>
      <machine maxCpus='255'>pc-i440fx-4.0</machine>
      <machine maxCpus='255'>pc-i440fx-5.0</machine>
      <machine maxCpus='255'>pc-i440fx-2.8</machine>
      <machine maxCpus='288'>pc-q35-6.2</machine>
      <machine maxCpus='255'>pc-i440fx-3.0</machine>
      <machine maxCpus='255'>pc-q35-2.5</machine>
      <machine maxCpus='255'>pc-i440fx-7.2</machine>
      <machine maxCpus='288'>pc-q35-2.11</machine>
      <domain type='qemu'/>
      <domain type='kvm'/>
    </arch>
    <features>
      <pae/>
      <nonpae/>
      <acpi default='on' toggle='yes'/>
      <apic default='on' toggle='no'/>
      <cpuselection/>
      <deviceboot/>
      <disksnapshot default='on' toggle='no'/>
    </features>
  </guest>

  <guest>
    <os_type>hvm</os_type>
    <arch name='x86_64'>
      <wordsize>64</wordsize>
      <emulator>/usr/bin/qemu-system-x86_64</emulator>
      <machine maxCpus='255'>pc-i440fx-8.1</machine>
      <machine canonical='pc-i440fx-8.1' maxCpus='255'>pc</machine>
      <machine maxCpus='288'>pc-q35-5.2</machine>
      <machine maxCpus='255'>pc-i440fx-2.12</machine>
      <machine maxCpus='255'>pc-i440fx-2.0</machine>
      <machine maxCpus='1'>xenpv</machine>
      <machine maxCpus='255'>pc-i440fx-6.2</machine>
      <machine maxCpus='288'>pc-q35-4.2</machine>
      <machine maxCpus='255'>pc-i440fx-2.5</machine>
      <machine maxCpus='255'>pc-i440fx-4.2</machine>
      <machine maxCpus='255'>pc-i440fx-5.2</machine>
      <machine maxCpus='255' deprecated='yes'>pc-i440fx-1.5</machine>
      <machine maxCpus='255'>pc-q35-2.7</machine>
      <machine maxCpus='288'>pc-q35-7.1</machine>
      <machine maxCpus='255'>pc-i440fx-2.2</machine>
      <machine maxCpus='1024'>pc-q35-8.1</machine>
      <machine canonical='pc-q35-8.1' maxCpus='1024'>q35</machine>
      <machine maxCpus='255'>pc-i440fx-2.7</machine>
      <machine maxCpus='288'>pc-q35-6.1</machine>
      <machine maxCpus='128'>xenfv-3.1</machine>
      <machine canonical='xenfv-3.1' maxCpus='128'>xenfv</machine>
      <machine maxCpus='255'>pc-q35-2.4</machine>
      <machine maxCpus='255'>pc-i440fx-7.1</machine>
      <machine maxCpus='288'>pc-q35-2.10</machine>
      <machine maxCpus='1'>x-remote</machine>
      <machine maxCpus='288'>pc-q35-5.1</machine>
      <machine maxCpus='255' deprecated='yes'>pc-i440fx-1.7</machine>
      <machine maxCpus='288'>pc-q35-2.9</machine>
      <machine maxCpus='255'>pc-i440fx-2.11</machine>
      <machine maxCpus='288'>pc-q35-3.1</machine>
      <machine maxCpus='255'>pc-i440fx-6.1</machine>
      <machine maxCpus='288'>pc-q35-4.1</machine>
      <machine maxCpus='255'>pc-i440fx-2.4</machine>
      <machine maxCpus='255'>pc-i440fx-4.1</machine>
      <machine maxCpus='255'>pc-i440fx-5.1</machine>
      <machine maxCpus='255'>pc-i440fx-2.9</machine>
      <machine maxCpus='1'>isapc</machine>
      <machine maxCpus='255' deprecated='yes'>pc-i440fx-1.4</machine>
      <machine maxCpus='255'>pc-q35-2.6</machine>
      <machine maxCpus='255'>pc-i440fx-3.1</machine>
      <machine maxCpus='288'>pc-q35-2.12</machine>
      <machine maxCpus='288'>pc-q35-7.0</machine>
      <machine maxCpus='255'>pc-i440fx-2.1</machine>
      <machine maxCpus='288'>pc-q35-8.0</machine>
      <machine maxCpus='255'>pc-i440fx-8.0</machine>
      <machine maxCpus='288'>pc-q35-6.0</machine>
      <machine maxCpus='255'>pc-i440fx-2.6</machine>
      <machine maxCpus='288'>pc-q35-4.0.1</machine>
      <machine maxCpus='255'>pc-i440fx-7.0</machine>
      <machine maxCpus='255' deprecated='yes'>pc-i440fx-1.6</machine>
      <machine maxCpus='288'>pc-q35-5.0</machine>
      <machine maxCpus='288'>pc-q35-2.8</machine>
      <machine maxCpus='255'>pc-i440fx-2.10</machine>
      <machine maxCpus='288'>pc-q35-3.0</machine>
      <machine maxCpus='288'>pc-q35-7.2</machine>
      <machine maxCpus='288'>pc-q35-4.0</machine>
      <machine maxCpus='128'>xenfv-4.2</machine>
      <machine maxCpus='255'>pc-i440fx-6.0</machine>
      <machine maxCpus='288'>microvm</machine>
      <machine maxCpus='255'>pc-i440fx-2.3</machine>
      <machine maxCpus='255'>pc-i440fx-4.0</machine>
      <machine maxCpus='255'>pc-i440fx-5.0</machine>
      <machine maxCpus='255'>pc-i440fx-2.8</machine>
      <machine maxCpus='288'>pc-q35-6.2</machine>
      <machine maxCpus='255'>pc-q35-2.5</machine>
      <machine maxCpus='255'>pc-i440fx-3.0</machine>
      <machine maxCpus='255'>pc-i440fx-7.2</machine>
      <machine maxCpus='288'>pc-q35-2.11</machine>
      <domain type='qemu'/>
      <domain type='kvm'/>
    </arch>
    <features>
      <acpi default='on' toggle='yes'/>
      <apic default='on' toggle='no'/>
      <cpuselection/>
      <deviceboot/>
      <disksnapshot default='on' toggle='no'/>
    </features>
  </guest>

</capabilities>
```
And full output of capabilities piped to cpu-baseline:
```
<cpu mode='custom' match='exact'>
  <model fallback='forbid'>EPYC-Rome</model>
  <vendor>AMD</vendor>
  <feature policy='require' name='3dnowprefetch'/>
  <feature policy='require' name='abm'/>
  <feature policy='require' name='adx'/>
  <feature policy='require' name='aes'/>
  <feature policy='require' name='amd-psfd'/>
  <feature policy='require' name='amd-ssbd'/>
  <feature policy='require' name='amd-stibp'/>
  <feature policy='require' name='apic'/>
  <feature policy='require' name='arat'/>
  <feature policy='require' name='auto-ibrs'/>
  <feature policy='require' name='avx'/>
  <feature policy='require' name='avx2'/>
  <feature policy='require' name='avx512-bf16'/>
  <feature policy='require' name='avx512-vpopcntdq'/>
  <feature policy='require' name='avx512bitalg'/>
  <feature policy='require' name='avx512bw'/>
  <feature policy='require' name='avx512cd'/>
  <feature policy='require' name='avx512dq'/>
  <feature policy='require' name='avx512f'/>
  <feature policy='require' name='avx512ifma'/>
  <feature policy='require' name='avx512vbmi'/>
  <feature policy='require' name='avx512vbmi2'/>
  <feature policy='require' name='avx512vl'/>
  <feature policy='require' name='avx512vnni'/>
  <feature policy='require' name='bmi1'/>
  <feature policy='require' name='bmi2'/>
  <feature policy='require' name='clflush'/>
  <feature policy='require' name='clflushopt'/>
  <feature policy='require' name='clwb'/>
  <feature policy='require' name='clzero'/>
  <feature policy='require' name='cmov'/>
  <feature policy='require' name='cmp_legacy'/>
  <feature policy='require' name='cr8legacy'/>
  <feature policy='require' name='cx16'/>
  <feature policy='require' name='cx8'/>
  <feature policy='require' name='de'/>
  <feature policy='require' name='decodeassists'/>
  <feature policy='require' name='erms'/>
  <feature policy='require' name='extapic'/>
  <feature policy='require' name='f16c'/>
  <feature policy='require' name='flush-l1d'/>
  <feature policy='require' name='flushbyasid'/>
  <feature policy='require' name='fma'/>
  <feature policy='require' name='fpu'/>
  <feature policy='require' name='fsgsbase'/>
  <feature policy='require' name='fxsr'/>
  <feature policy='require' name='fxsr_opt'/>
  <feature policy='require' name='gfni'/>
  <feature policy='require' name='ht'/>
  <feature policy='require' name='ibpb'/>
  <feature policy='require' name='ibrs'/>
  <feature policy='require' name='ibs'/>
  <feature policy='require' name='invpcid'/>
  <feature policy='require' name='invtsc'/>
  <feature policy='require' name='lahf_lm'/>
  <feature policy='require' name='lbrv'/>
  <feature policy='require' name='lfence-always-serializing'/>
  <feature policy='require' name='lm'/>
  <feature policy='require' name='mca'/>
  <feature policy='require' name='mce'/>
  <feature policy='require' name='misalignsse'/>
  <feature policy='require' name='mmx'/>
  <feature policy='require' name='mmxext'/>
  <feature policy='require' name='monitor'/>
  <feature policy='require' name='movbe'/>
  <feature policy='require' name='msr'/>
  <feature policy='require' name='mtrr'/>
  <feature policy='require' name='no-nested-data-bp'/>
  <feature policy='require' name='npt'/>
  <feature policy='require' name='nrip-save'/>
  <feature policy='require' name='null-sel-clr-base'/>
  <feature policy='require' name='nx'/>
  <feature policy='require' name='osvw'/>
  <feature policy='require' name='pae'/>
  <feature policy='require' name='pat'/>
  <feature policy='require' name='pause-filter'/>
  <feature policy='require' name='pclmuldq'/>
  <feature policy='require' name='pdpe1gb'/>
  <feature policy='require' name='perfctr_core'/>
  <feature policy='require' name='perfctr_nb'/>
  <feature policy='require' name='pfthreshold'/>
  <feature policy='require' name='pge'/>
  <feature policy='require' name='pku'/>
  <feature policy='require' name='pni'/>
  <feature policy='require' name='popcnt'/>
  <feature policy='require' name='pse'/>
  <feature policy='require' name='pse36'/>
  <feature policy='require' name='rdpid'/>
  <feature policy='require' name='rdrand'/>
  <feature policy='require' name='rdseed'/>
  <feature policy='require' name='rdtscp'/>
  <feature policy='require' name='sep'/>
  <feature policy='require' name='sha-ni'/>
  <feature policy='require' name='skinit'/>
  <feature policy='require' name='smap'/>
  <feature policy='require' name='smep'/>
  <feature policy='require' name='sse'/>
  <feature policy='require' name='sse2'/>
  <feature policy='require' name='sse4.1'/>
  <feature policy='require' name='sse4.2'/>
  <feature policy='require' name='sse4a'/>
  <feature policy='require' name='ssse3'/>
  <feature policy='require' name='stibp-always-on'/>
  <feature policy='require' name='svm'/>
  <feature policy='require' name='svm-lock'/>
  <feature policy='require' name='svme-addr-chk'/>
  <feature policy='require' name='syscall'/>
  <feature policy='require' name='tce'/>
  <feature policy='require' name='topoext'/>
  <feature policy='require' name='tsc'/>
  <feature policy='require' name='tsc-scale'/>
  <feature policy='require' name='umip'/>
  <feature policy='require' name='v-vmsave-vmload'/>
  <feature policy='require' name='vaes'/>
  <feature policy='require' name='vgif'/>
  <feature policy='require' name='vmcb-clean'/>
  <feature policy='require' name='vme'/>
  <feature policy='require' name='vnmi'/>
  <feature policy='require' name='vpclmulqdq'/>
  <feature policy='require' name='wbnoinvd'/>
  <feature policy='require' name='wdt'/>
  <feature policy='require' name='x2apic'/>
  <feature policy='require' name='xgetbv1'/>
  <feature policy='require' name='xsave'/>
  <feature policy='require' name='xsavec'/>
  <feature policy='require' name='xsaveerptr'/>
  <feature policy='require' name='xsaveopt'/>
  <feature policy='require' name='xsaves'/>
</cpu>
```

**Comment by karmab** [comment]
this should be addressed by https://github.com/karmab/kcli/commit/01ed4bb5df82eb52782c2c553dc0876b0342ef0e
please confirm ! 💯

**Comment by wainersm** [comment]
Hi @karmab !

> this should be addressed by [01ed4bb](https://github.com/karmab/kcli/commit/01ed4bb5df82eb52782c2c553dc0876b0342ef0e) please confirm ! 💯

Let me test it. thanks!

**Comment by wainersm** [comment]
> Hi @karmab !
> 
> > this should be addressed by [01ed4bb](https://github.com/karmab/kcli/commit/01ed4bb5df82eb52782c2c553dc0876b0342ef0e) please confirm ! 💯
> 
> Let me test it. thanks!

And it worked out here, @karmab . Nested virt working as expected.
